spec/lib/extensions/ar_migration_spec.rb

```diff
@@ -1,6 +1,7 @@
 describe ArPglogicalMigrationHelper do
   shared_context "without the schema_migrations_ran table" do
     before do
+      allow(ActiveRecord::SchemaMigration).to receive(:table_name).and_return("schema_migrations_ran")
```

Comment:
This is wrong. `ActiveRecord::SchemaMigration` is a different thing than the `schema_migrations_ran` table.

```
irb(main):004:0> ActiveRecord::SchemaMigration.table_name
=> "schema_migrations"
```

Is the issue that `ActiveRecord::SchemaMigration` calls `connection.table_exists?`?
Previously we could stub a method with particular arguments and that would not interfere with other calls to that method with different arguments. Is that what rspec is complaining about?

Reply:
The error says this "Please stub a default value first if message might be received with other args as well."

How do we ensure that the "default value" is calling the original method? I thought that was implied.

Reply:
If we change this line to `allow(ActiveRecord::Base.connection).to receive(:table_exists?).and_call_original` and leave the next line as is, will that fix the problem?

Reply:
@carbonin Yes, that works. I'll update it and replace what I had with that.